doc/plugin_server_upstreamca_disk.md

```diff
@@ -5,6 +5,9 @@ intermediate signing certificates for the server's signing authority. The
 intermediate certificates are minted against CSRs generated by the ServerCA
 plugin.
 
+The `disk` plugin reloads root CA credentials on all CSR requests. If the
+credentials cannot be loaded, then the previously loaded credentials will be used.
```

Comment:
It would be nice to explain the rationale for this, for a naive reader.

Reply:
Can do! I didn't want to add too much bulk before the table that describes the config options, what do you think?

I feel that if I explain this after the table, it's less likely to be read. Maybe that is ok.